Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/model/GameDatabase.java

```diff
@@ -262,33 +262,29 @@ else if (!folder.exists())
 
 	public Observable<Cursor> getGamesForPlatform(final Platform platform)
 	{
-		return Observable.create(new Observable.OnSubscribe<Cursor>()
-		{
-			@Override
-			public void call(Subscriber<? super Cursor> subscriber)
-			{
-				Log.info("[GameDatabase] Reading games list...");
-
-				String[] whereArgs = new String[]{Integer.toString(platform.toInt())};
-
-				SQLiteDatabase database = getReadableDatabase();
-				Cursor resultCursor = database.query(
-						TABLE_NAME_GAMES,
-						null,
-						KEY_GAME_PLATFORM + " = ?",
-						whereArgs,
-						null,
-						null,
-						KEY_GAME_TITLE + " ASC"
-				);
-
-				// Pass the result cursor to the consumer.
-				subscriber.onNext(resultCursor);
-
-				// Tell the consumer we're done; it will unsubscribe implicitly.
-				subscriber.onCompleted();
-			}
-		});
+		return Observable.create(subscriber ->
+        {
+            Log.info("[GameDatabase] Reading games list...");
```

Comment:
The indentation levels here are weird... Are some lines using tabs and some lines using spaces?